src/bundle/Resources/config/services/normalizers.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    ibexa.search.suggestion.serializer:
+        class: Symfony\Component\Serializer\Serializer
+        autoconfigure: false
+        arguments:
+            $normalizers:
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer'
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\LocationNormalizer'
+            $encoders:
+                - '@serializer.encoder.json'
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer:
+        autoconfigure: false
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationCollectionNormalizer:
+        autoconfigure: false
```

Comment:
What do you intend to do with this normalizer? You are not adding it to normalizer list for the search serializer above, so it does not participate anywhere.

Note that your description in PR states that tag will be working. For clarification remove this part of description, or apply it.